test-cljs/com/climate/geojson_schema/test/resources.clj

```diff
@@ -0,0 +1,16 @@
+(ns com.climate.geojson-schema.test.resources
+  (:require [clojure.java.io :refer [resource file]]
+            [cheshire.core :as json]))
+
+(defmacro all-resources
+  "Eagerly pulls in all of the geojson_examples.
+  Pass in the symbol to def in code.
+  It will contain a map short-file -> file-data"
+  [symbol]
+  (let [data (->> (resource "geojson_examples")
+                  file
+                  file-seq
+                  (remove #(.isDirectory %))
+                  (map (juxt #(.getName %) #(json/parse-string (slurp %) true)))
+                  (into {}))]
+    `(def ~symbol ~data)))
```

Comment:
It's not possible to read resources in the same way within Clojurescript (the `clojure.java` namespaces won't work); by creating a macro in Clj that pre-defines the value for Cljs is a general work around 
See [http://stackoverflow.com/questions/23767729/clojurescript-templating-from-html-files](http://stackoverflow.com/questions/23767729/clojurescript-templating-from-html-file) for some more explanation.

Instead of `(def ~symbol ~data)` yes it might be cleaner to return `~data` so that the call becomes `(def data (all-resources))`